packages/backend/src/payment-method/ilp/connector/core/middleware/error-handler.ts

```diff
@@ -23,7 +23,7 @@ export function createIncomingErrorHandlerMiddleware(
       if (!err || typeof err !== 'object') {
         err = new Error('Non-object thrown: ' + e)
       }
-      ctx.services.logger.debug({ err }, 'Error thrown in incoming pipeline')
+      ctx.services.logger.info({ err }, 'Error thrown in incoming pipeline')
```

Comment:
I figured we don't want to limit the error logs to development so I changed to info (which matches what we do in the open payments server error handler as well.)

Reply:
Maybe we should have an error log in the unexpected internal error case down below?

Reply:
there is one but it looks like its basically null if its not an `HttpError`. changed it to always log with the original error.